src/main/java/org/folio/inventory/resources/Items.java

```diff
@@ -107,6 +115,57 @@ public void register(Router router) {
           routingContext, webContext));
   }
 
+  private void move(RoutingContext routingContext) {
+
+    WebContext context = new WebContext(routingContext);
+    JsonObject itemsMoveJsonRequest = routingContext.getBodyAsJson();
+
+    Optional<ValidationError> validationError = itemsMoveHasRequiredFields(itemsMoveJsonRequest);
+    if (validationError.isPresent()) {
+      unprocessableEntity(routingContext.response(), validationError.get());
+    } else {
+      ItemsMove itemsMove = MoveUtil.jsonToItemsMove(itemsMoveJsonRequest);
+      List<String> itemIdsToUpdate = itemsMove.getItemIds();
+      try {
+        storage.getItemCollection(context).findByCql(
```

Comment:
This client is not safe for fetching records using a long CQL query. It causes `414 URI Too Long` error when with > ~40-50 UUIDs in query. Please use `org.folio.inventory.storage.external.MultipleRecordsFetchClient` instead.

Reply:
Done